README.md

```diff
@@ -161,6 +161,14 @@ Tell the player to show a placeholder that indicates an error.
 
 Example: `playerApi.error(new Error('Something bad happened to my gadget'))`
 
+#### `recordActivity`
```

Comment:
Do we have any constraints for this? e.g. should `data` have any specific shape, required attrs, etc?